src/jq_test.c

```diff
@@ -490,4 +490,98 @@ static void jv_test() {
     //jv_dump(jv_copy(o2), 0); printf("\n");
     jv_free(o2);
   }
+
+  {
+	assert(jv_equal(
+			jv_paths(
```

Comment:
Why do we want to use an `assert()` with `jv_paths()`?  Why not just assert that the two values are equal?

Reply:
(I'm not saying I don't want `jv_paths()`, I'm just saying that I think we don't need `jv_paths()` for testing `jv_unshared()`, so `jv_paths()` -if we need it at all- can be in a separate PR.

Reply:
I just did it out of convenience since I iterate through every path in the unshared value to see whether it really has no shared memory

Reply:
It would be faster to just recurse in `jv_is_unshared()`.